app/views/migration/_form.html.erb

```diff
@@ -8,7 +8,7 @@
           <%= contributor.input(:foaf_mbox, as: :email, required: true) %>
         <% end %>
         <%= cho.association(:edm_wasPresentAt, include_blank: :translate) %>
-        <%= cho.input(:dc_identifier, input_html: { 'data-requires': 'ore_aggregation_edm_aggregatedCHO_attributes_edm_wasPresentAt_id' }) %>
+        <%= cho.input(:dc_identifier, wrapper_html: {class:'requires-other-field'}, input_html: { 'data-requires': 'ore_aggregation_edm_aggregatedCHO_attributes_edm_wasPresentAt_id' }) %>
```

Comment:
This seems superfluous: it should be derivable from the presence of the `data-requires` attribute.